README

```diff
@@ -1,3 +1,17 @@
+Installation
+============
+
+In a Python virtual environment to avoid conflicts with packages from the
+distribution's package manager:
+
+$ python3 -m venv myvenv
+$ source myvenv/bin/activate
+$ pip install juniper-vpn-py
+
+Once that venv activated, the "juniper-vpn.py" binary will be in the path.
+NOTE: DO NOT USE ./juniper-vpn.py shim distributed in the git repository, use
```

Comment:
It does work, but that would do a system wide install. This means that pip will go and write in the site-packages directory, e.g. /usr/lib/python3.8/site-packages. The problem with that is that the exact same folder is used to install packages using the distro package manager like apt. 

That means that pip will override files managed by the distro package manager, potentially updating a package to a version that will break other installed software packaged by your distro. If you pass --user to pip, you will install in the HOME but otherwise the issue will remain the same as the user-local site-packages directory takes precedence (as seen in `sys.path`).

The only foolproof way of avoiding that is to use a venv/virtualenv so that the installation is completely isolated and will not conflict with anything else on the system. 

That means the following:
* system-wide `pip install` should only be used to create distro packages, or for someone ready to deal with the breakage that will follow its use (on some very up-to-date systems like Archlinux, it's not too much of an issue as the distro packages the latest version anyway). Also, system wide `pip install` will need to be ran as root although it might trigger complex work including building software.
* `pip install --user` is to be treated the same way, although the mess is easier to cleanup
* `venv` (or `virtualenv`) is the only trustworthy recommendation that can be done when it's supposed to be distribution-agnostic .

Reply:
OK, let's stick with that then. Just seems a bit odd to fire up a virtualenv just to connect to the VPN :)

Reply:
Fortunately that's straightforward and easy to setup. I've come across insane setups based on wild use of PYTHONPATH, which is most of the time a recipe for endless pain ...